authors/apps/authentication/serializers.py

```diff
@@ -190,7 +195,6 @@ class Meta:
         # password field, we needed to specify the `min_length` and
         # `max_length` properties too, but that isn't the case for the token
         # field.
-        read_only_fields = ('token',)
```

Comment:
why did you remove this?

Reply:
This was no longer needed code, it  use was that it was necessary when displaying the token along with user information